packages/toolpad-core/src/types.ts

```diff
@@ -84,25 +84,25 @@ export interface FunctionValueType extends ValueTypeBase {
 
 export interface ArgControlSpec {
   type:
-  | 'boolean' // checkbox
-  | 'number' // number input
-  | 'range' // slider
-  | 'object' // json editor
-  | 'radio' // radio buttons
-  | 'buttons' // button group
-  | 'select' // select control
-  | 'string' // text input
-  | 'color' // color picker
-  | 'slot' // slot in canvas
-  | 'slots' // slots in canvas
-  | 'multiSelect' // multi select ({ type: 'array', items: { type: 'enum', values: ['1', '2', '3'] } })
-  | 'date' // date picker
-  | 'json' // JSON editor
-  | 'GridColumns' // GridColumns specialized editor
-  | 'HorizontalAlign'
-  | 'VerticalAlign'
-  | 'function'
-  | 'RowIdFieldSelect'; // Row id field specialized select
+    | 'boolean' // checkbox
```

Comment:
Is it your editor prettier extension doing that? If so, let's debug that next week

Reply:
maybe it's something in my global vscode settings, i'll take a look